lib/blocks/babel.js

```diff
@@ -0,0 +1,44 @@
+module.exports = blockConfig => (processEnv, argv) => config => {
+    const isDev = processEnv.NODE_ENV === "development";
+    const isHotReload = processEnv.HOT_RELOAD === "true";
+
+    const defaultConf = {
+        useCache: isDev,
+        development: isDev,
+        hot: isHotReload,
+        test: /\.(js|jsx)$/
+    };
+
+    const mergedConf = { ...defaultConf, ...(blockConfig || {}) };
+
+    config.module.rules.push({
+        test: mergedConf.test,
+        exclude: /node_modules/,
```

Comment:
I think we need a way to keep this customizable, eg: mwc doesn't export by default es5 code

Reply:
Sure thing.